blueocean-dashboard/src/main/js/components/records.jsx

```diff
@@ -65,6 +69,15 @@ export const RunsRecord = Record({
     pullRequest: PullRequestRecord,
 });
 
+export class FolderRecord extends Record({
```

Comment:
I guess that's where I was at. It's not the end of the world to add the props for num of folders and pipelines but it makes it less clear to what is what. What if the Record for Pipeline had an `isFolder()` method? That would at least a bit more clear.